B827EBFFFF46EC6D.json

```
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFF46EC6D",
+    "servers": [
+      {
+        "server_address": "router.us.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": "-27.571021",
```

The numbers should be numbers, not strings.
```suggestion
    "ref_latitude": -27.571021,
```